### PR TITLE
docs: logo customization

### PR DESCRIPTION
**Pull request by jgngo**
Added documentation on how to customize the logo.

### CODE REVIEW

File: .docs/content/1.introduction/4.configuration.md [hunk 1]
```diff
@@ -130,3 +130,17 @@ export default defineAppConfig({
...
+
+To update the logo in the header, create a vue component with your own logo. In this example, the image is located at `/public/img`.
+
+```
+/components/Logo.vue
+```
+
+```
+<template>
+    <img width="120" src="/img/YOURLOGO.png"/>
```
Comment (Atinux) [inline .docs/content/1.introduction/4.configuration.md:144]:
```suggestion
  <img width="120" src="/img/YOURLOGO.png"/>
```

File: .docs/content/1.introduction/4.configuration.md [hunk 2]
```diff
@@ -130,3 +130,17 @@ export default defineAppConfig({
 | `github.repo`                | `string`   |                  | Name of the GitHub repo to edit files                                                                |
 | `github.owner`               | `string`   |                  | Owner of the repo                                                                                    |
 | `github.edit`                | `boolean`  |                  | Toggle "Edit this page on Github" component on documentation pages                                   |
+
+## Customizing the logo
+
+To update the logo in the header, create a vue component with your own logo. In this example, the image is located at `/public/img`.
```
Comment (Atinux) [inline .docs/content/1.introduction/4.configuration.md:136]:
```suggestion
To update the logo in the header, create a component in `components/Logo.vue` with your own logo.

In this example, the image is located at `/public/img`.
```

File: .docs/content/1.introduction/4.configuration.md [hunk 3]
```diff
@@ -130,3 +130,17 @@ export default defineAppConfig({
...
+
+```
+/components/Logo.vue
+```
+
```
Comment (Atinux) [inline .docs/content/1.introduction/4.configuration.md:141]:
```suggestion
```

File: .docs/content/1.introduction/4.configuration.md [hunk 4]
```diff
@@ -130,3 +130,15 @@ export default defineAppConfig({
...
 | `github.owner`               | `string`   |                  | Owner of the repo                                                                                    |
 | `github.edit`                | `boolean`  |                  | Toggle "Edit this page on Github" component on documentation pages                                   |
+
+## Customizing the logo
+
+To update the logo in the header, create a component in `components/Logo.vue` with your own logo.
+
+In this example, the image is located at `/public/img`.
+
+```
```
Comment (Atinux) [inline .docs/content/1.introduction/4.configuration.md:140]:
```suggestion
```vue [components/Logo.vue]
```

**Review by Atinux — APPROVED** [review]

**Comment by Tahul** [comment]
Nice work; thanks a lot!

If you spot other possible improvements from the documentation, I would he happy to merge them!

**Pull request merged**